docs/ref/files/file.txt

```diff
@@ -153,6 +154,17 @@ below) will also have a couple of extra methods:
     :class:`File` or of a subclass of :class:`File`, such as
     :class:`~django.core.files.base.ContentFile`.
 
+    This method will not pass any additional arguments like ``update_fields``
+    to the model's :meth:`~django.db.models.Model.save` method. To provide
+    additional arguments to the model's :meth:`~django.db.models.Model.save`
+    method, set ``save`` to ``False`` and then call the model's ``save()``
+    method directly:
+
+    .. code-block:: pycon
+
+        >>> car.photo.save("myphoto.jpg", content, save=False)
+        >>> car.save(update_fields=["photo"])
+
```

Comment:
Oh I didn't realise this section already declared the 2 `save()` methods, which kind of almost (albeit indirectly) gets the point across anyway.

My 2¢ is that this section is too much and should probably be folded into the previous section as a note or something similar. Reference documentation needs to have just the right amount of signal/noise, folks shouldn't need to read through paragraphs to pick up the essential points which are:
 - save(save=True) calls the model save
 - (this is already implicit but won't hurt to add a footnote to make it explicit) it doesn't pass through update_fields so that needs to be done manually

Reply:
Fair enough, thanks for looking at it.